src/components/app/components/Navbar.js

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Avatar,  Dropdown, Layout, Menu } from 'antd';
+
+import AppLink from './AppLink';
+import home from '../../home';
+
+import signin from '../../signin';
+import signup from '../../signup';
+import { LOGO, MARKET_TEXT, NAVBAR, PATHS, USER_AVATAR } from '../constants';
+
+const { Item } = Menu;
+const { Header } = Layout;
+const { AVATAR, SHAPE, SIZE, SOURCE } = LOGO;
+const { MAIN_NAV, NAV_MENU, NAV_MODE, NAV_THEME } = NAVBAR;
+const { CONTACT, HOME, MARKET, PROFILE, SIGN_IN, SIGN_UP } = PATHS;
+const { CONTAINER, ICON_TYPE, SIGN_OUT, USER_DROP_DOWN, USER_PROFILE } = USER_AVATAR;
+
+const { ContactUs } = home.components;
+const { Signin } = signin.components;
+const { Signup } = signup.components;
+
+const UserMenu = (
+    <Menu>
+        <Item key={USER_PROFILE}>
+            <AppLink to={PROFILE} key={PROFILE}>{USER_PROFILE}</AppLink>
+        </Item>
+        <Item key={SIGN_OUT}>{SIGN_OUT}</Item>
+    </Menu>
+);
+
+const items = [
```

Comment:
is lines 32 to 40 a array of components or what?

Reply:
Correct, it is an array of components

Reply:
ok